webpages/settings/components/addon-body.html

```diff
@@ -369,8 +368,8 @@
     position: relative;
     cursor: pointer;
     transition: all 0.25s ease;
-    border: 1px solid var(--switch-border);
```

Comment:
Changing the button's colour and removing the border has unintentionally reverted the switch part of #6240.